pydsa/binary_search.py

```diff
@@ -0,0 +1,59 @@
+def binary_search(items_list, item, recursive=False, first_ptr = -1, last_ptr = -1):
```

Comment:
Please follow PEP8 Standards:
pep8: E251 remove space around `=`
i.e. `first_ptr = -1` --> `first_ptr=1` & `last_ptr = -1` --> `last_ptr=-1`

Tip: there is a plugin for checking pep8 in every code editor, checkout for your code editor.